src/mesh/mesh_refinement.cpp

```diff
@@ -129,6 +175,7 @@ void MeshRefinement::RestrictCellCenteredValues(const ParArrayND<Real> &fine,
                (fine(n, 0, j, i + 1) * vol01 + fine(n, 0, j + 1, i + 1) * vol11)) /
               tvol;
         });
+    applyBounds(pmb, coarse, cib, cjb);
```

Comment:
Why do we only have a function like this for the 2d case?

Reply:
good catch. I think this should be for all 3 cases, @nmsriram

Reply:
Oh brother. yes.  of course.  @JoshuaSBrown / @Yurlungur  Should I put in another PR to fix this, or should I just depend on @Yurlungur to fix it?

Reply:
Whoever, gets to it first I suppose.